File: bench/sockets/1.go
package main

import (
	"bufio"
	"io"
	"net"
	"fmt"
	"time"
)

func main() {
	// Listen
	var ln, _ = net.Listen("tcp", ":8080")

	// Accept connection
	var conn, _ = ln.Accept()

	var start = time.Now()

	// Run loop forever
	for {
		// Will listen for message to process ending in newline (\n)
		var _, err = bufio.NewReader(conn).ReadString('\n')
		if err == io.EOF {
			return
		}

		// Reply
		var newmessage = "Hello from server\n"
		fmt.Fprintf(conn, newmessage)
	}

	fmt.Println(time.Now().Sub(start).Nanoseconds())
}

File: bench/hello-world/1.cpp
#include <iostream>
#include <chrono>

using namespace std;

int main()
{
    auto start = chrono::steady_clock::now();

	cout << "Hello, world!" << endl;

    cout << chrono::duration_cast<chrono::nanoseconds>(chrono::steady_clock::now() - start).count() << endl;

	return 0;
}
